CMakeLists.txt

```diff
@@ -117,6 +117,13 @@ add_subdirectory(mavis)
 include_directories(SYSTEM softfloat)
 add_subdirectory(softfloat)
 
+# STF Library
+set(DISABLE_STF_DOXYGEN 1)
```

Comment:
RE: https://github.com/sparcians/stf_lib/pull/52,

You can avoid the name collision for STF by adding:
```
set(DISABLE_STF_TESTS 1)
```